### PR TITLE
[master]re-enable deb publishing

### PR DESCRIPTION
**Pull request by brthor**
@eerhardt @piotrpMSFT

### CODE REVIEW

**Comment by brthor** [comment]
@dotnet-bot test  OSX x64 Release Build please

File: build_projects/shared-build-targets-utils/Utils/DebPackageCreator.cs [hunk 1]
```diff
@@ -111,9 +111,8 @@ private void CreateEmptyDebianLayout(DebianLayoutDirectories layoutDirectories)
             foreach (var variable in debianConfigVariables)
             {
                 var variableToken = $"%{variable.Key}%";
-                debianConfigFileContents.Replace(variableToken, variable.Value);
+                debianConfigFileContents = debianConfigFileContents.Replace(variableToken, variable.Value);
```
Comment (livarcocc) [inline build_projects/shared-build-targets-utils/Utils/DebPackageCreator.cs]: You could use a string builder here and then you wouldn't have to keep doing this ugly assignment.

**Comment by livarcocc** [comment]
:shipit: once it passes CI

**Comment by brthor** [comment]
will revisit this PR later today.

**Comment by brthor** [comment]
Closing as #184 covers bringing over the changes needed to make this work

**Pull request closed without merging**